sources/packages/backend/libs/integrations/src/services/student/student.service.ts

```diff
@@ -145,6 +106,50 @@ export class StudentService extends RecordDataModelService<Student> {
     return studentRepo.save(studentToUpdate);
   }
 
+  /**
+   * Get student by SIN, Last name and birth date.
+   * @param sin sin.
+   * @param lastName last name.
+   * @param birthDate birth date.
+   * @returns student.
+   */
+  async getStudentBySINAndLastNameAndBirthDate(
```

Comment:
IMO just a suggestion, the method name can be called as findStudentByPersonalInfo or getStudentByPersonalInfo. seems to be a big method name with 2 ANDs